UnitTests/PayPalTests/PayPalClient_Tests.swift

```diff
@@ -0,0 +1,72 @@
+import XCTest
+import PayPalCheckout
+@testable import PaymentsCore
+@testable import PayPal
+
+class PayPalClient_Tests: XCTestCase {
```

Comment:
Do we have any reason to separate the Tests word with an underscore?

Reply:
It makes sifting through a soon-to-be very large suite of test files a lot easier visually. So, solely for readability & organization.